src/components/TodoOutput/TodoOutputList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TodoOutputList = () => {
+  return (
+    <div className="list">
```

Comment:
is there reason for using class name "list"?  if you use list, i thought `<ul> `is better